README.md

```diff
@@ -50,12 +50,21 @@ The plugin requires some configuration to connect to Vault. The parameters are:
 | VAULT_ADDR     | Address of your Vault      | N/A                  |
 | PATH_PREFIX    | Prefix of the vault path to look for the secrets | N/A       |
 | TYPE           | The type of Vault backend  | Supported values: `vault` and `secretmanager` |
+| KV_VERSION    | The vault secret engine  | Supported values: `1` and `2` (defaults to 2)|
 | AUTH_TYPE      | The type of authentication | Supported values: vault: `approle, github`   secretmanager: `iam` |
 | GITHUB_TOKEN   | Github token               | Required with `AUTH_TYPE` of `github` |
 | ROLE_ID        | Vault AppRole Role_ID      | Required with `AUTH_TYPE` of `approle` |
 | SECRET_ID      | Vault AppRole Secret_ID    | Required with `AUTH_TYPE` of `approle` |
 | IBM_API_KEY    | IBM Cloud IAM API Key      | Required with `TYPE` of `secretmanager` and `AUTH_TYPE` of `iam` |
 
+#### Supported Annotation
+We support two different annotations that can be used inside a kubernetes resource. These annotations will override any corrisponding configuration set via Environment Variable or Configuration File.
+
+| Annotation | Description |  
+| ---------- | ----------- |  
+| avp_path | Path to the Vault Secret |
+| kvVersion | Version of the KV Secret Engine |
```

Comment:
I'd almost say we should decide on under_case or camelCase and be consistent, but idk if that really matters

Reply:
Hmm good call